contracts/limitOrder/LimitOrderBook.sol

```diff
@@ -112,13 +113,21 @@ contract LimitOrderBook is
         if (order.reduceOnly) {
```

Comment:
What about merging this if block with the other `if (order.reduceOnly)` block in line `145`? 
It's cleaner and I think an order will only be filled after a bot `callStatic` and the tx doesn't revert!